example/src/redux/reducer/login.js

```diff
@@ -18,30 +18,21 @@ export default function loginReducer (state = initialState, action = {}) {
       return {
         ...state,
         loading: false,
-        loggedIn: true
+        loggedIn: true,
+        user: action.user
       }
     case types.LOGIN.FAILURE:
       return {
         ...state,
         loading: false
       }
     case types.LOGOUT.SUCCESS:
-      return {
-        ...state,
-        loading: false,
-        loggedIn: false
-      }
+      return initialState
```

Comment:
Shouldn't we set `loggedIn` to `false` and `user` to `null` here?

Reply:
I thought this was doing that, as
```js
const initialState = {
  loading: false,
  loggedIn: false,
  user: null
}
```
I could be wrong though. I had problems getting the code to run locally, so wasn't able to test it.

Reply:
Oh yes, my bad, I read that too quickly. Looks good, thanks again. :)